spot/entrypoints/content/SavingControls.tsx

```diff
@@ -80,6 +80,10 @@ function SavingControls({ onClose, getVideoData, getErrorEvents }: ISavingContro
     onCleanup(() => window.removeEventListener("keydown", spacePressed));
   });
 
+  const handleNameChange = (inputValue: string) => {
```

Comment:
no point in redeclaring the same function here

Reply:
I initially started with a bit extended functionality. I've made it inline now.